internal/builder/packages.go

```diff
@@ -290,19 +293,43 @@ func signZippedPackage(options BuildOptions, zippedPackagePath string) error {
 	return nil
 }
 
-func copyLicenseTextFile(licensePath string) error {
-	_, err := os.Stat(licensePath)
-	if err == nil {
+// copyLicenseTextFile checks if a license file exists in the package directory.
+// If the file already exists in the package, it does nothing.
+// If the file does not exist in the package, it looks for a license file in the repository root directory.
+// If a license file is found in the repository, it copies it to the package directory.
+func copyLicenseTextFile(repoRoot *os.Root, licensePath string) error {
+	// if the given path exist, skip copying
+	info, err := os.Stat(licensePath)
+	if err == nil && !info.IsDir() {
 		logger.Debug("License file in the package will be used")
 		return nil
 	}
+	// if the given path does not exist, continue
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
It should not be needed to check `err != nil` , `errors.Is()` already takes that into account:

```suggestion
	if !errors.Is(err, os.ErrNotExist) {
```